llvm/test/CodeGen/RISCV/rvv/sink-splat-operands.ll

```diff
@@ -22,7 +22,7 @@ entry:
   %broadcast.splat = shufflevector <4 x i32> %broadcast.splatinsert, <4 x i32> poison, <4 x i32> zeroinitializer
   br label %vector.body
 
-vector.body:                                      ; preds = %vector.body, %entry
+vector.body:
```

Comment:
Same here for these comments, I think these are obscuring the actual code related changes from the diff

Reply:
you nean delete it or not?